andela/travel_tool_front
# #162006041 Fix room dropdown input field

#### What does this PR do?
This PR enables the Room drop down field to be prefilled with `No beds found` when editing a multicity request with a trip that has no rooms added for the destination.

#### Description of Task to be completed?
- Autofill room field with `No beds found` for trips with no rooms while editing request

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Change into the project directory -` cd travel_tool_front`
- Pull and checkout into this branch - `bg-fix-bed-input-field-162006041 `
- Install project dependencies using `yarn install`
- Clone the [Travela backend](https://github.com/andela/travel_tool_back)
- Run database migrations using `yarn run db:rollmigrate`
- Check `.env.example` and set up your `.env` appropriately
- Start the Server and Frontend with `make start` and log into the application
- Create a new multi-city travel request with at least one trip to a destination with no beds/rooms
- Navigate to `Edit` the request


#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[162006041](https://www.pivotaltracker.com/story/show/162006041)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/30747298/49221744-0ca7fc80-f3eb-11e8-9c34-6f1b5ad37489.png)



#### Questions:
N/A
